omod-2.4/src/main/java/org/openmrs/module/webservices/rest/web/resource/openmrs2_4/ServerLogResource2_4.java

```diff
@@ -7,7 +7,7 @@
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_4;
+package org.openmrs.module.webservices.rest.web.resource.openmrs2_4;
```

Comment:
do we need an importation only for this file?

Reply:
This has been addressed in the latest push, turns out I had miss read the package names.